packages/amesos2/src/Amesos2_EpetraRowMatrix_AbstractMatrixAdapter_decl.hpp

```diff
@@ -164,10 +169,12 @@ namespace Amesos2 {
 
     bool isGloballyIndexed_impl() const;
 
+    bool isGIDContiguous_impl() const;
+
```

Comment:
Not used, was added and used before I updated the solver parameter list to pass IsContiguous setting; will remove